internal/common/enums.rs

```diff
@@ -366,16 +366,16 @@ macro_rules! for_each_enums {
                 TabList,
                 /// The element is a container for tab content.
                 TabPanel,
-                /// The role for a `Text` element. It's automatically applied.
+                /// The role for a `Text` element. This is automatically applied to `Text` elements.
                 Text,
                 /// The role for a `TableView` or behaves like one.
                 Table,
                 /// The role for a TreeView or behaves like one. (Not provided yet)
                 Tree,
                 /// The element is a `ProgressIndicator` or behaves like one.
                 ProgressIndicator,
-                /// The role for widget with editable text such as a
-                /// `LineEdit` or a `TextEdit`
+                /// The role for widget with editable text such as a `LineEdit` or a `TextEdit`.
+                /// This is automatically applied to `TextInput`, `LineEdit`, and `TextEdit` elements.
```

Comment:
Ditto, probably not needed for widgets or we need to document it for every widgets